rfcs/text/0005-host-metric-fields.md

```diff
@@ -26,6 +26,17 @@ Proposed 7 new fields are:
 <!--
 Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
 -->
+This RFC calls for the addition of host fields to collect basic monitoring metrics from a host or VM such as CPU, network and disk.
+
+| field | type | description |
+| --- | --- | --- |
+| `host.cpu.usage` | scaled_float | Percent CPU used with scaling_factor of 1000. This value is normalized by the number of CPU cores and it ranges from 0 to 1. For example: For a two core host, this value should be the average of the 2 cores, between 0 and 1. |
+| `host.network.in.bytes` | long | The number of bytes received on all network interfaces by the host in a given period of time. |
+| `host.network.in.packets` | long | The number of packets received on all network interfaces by the host in a given period of time. |
+| `host.network.out.bytes` | long | The number of bytes sent out on all network interfaces by the host in a given period of time. |
```

Comment:
We have in ES now `_meta` which also support `byte` as unit: https://github.com/elastic/elasticsearch/pull/61941 Even though it is unfortunate that in older metrics we still have the postfix of `.bytes`, I think all new values we add we should skip the unit postfix and use `_meta` instead. 

@webmat ECS should add support for _meta.

Reply:
This is a good point, and I think we should avoid postfixes just for the sake of making the unit explicit. In the case of `system.network.out` object (this metric comes from it) this is what we have today:

```
    "system": {
        "network": {
            "in": {
                "bytes": 37904869172,
                "dropped": 32,
                "errors": 0,
                "packets": 32143403
            }
        } 
     }
```

I wonder how we should differentiate all the different metrics there, specially between bytes and packets

Reply:
Ha, interesting case. I'm tempted to say in this context the unit name might make sense. In any case, we should still add the info to _meta as this is what should be used by Kibana.

Alternative `bytes` could be called `data` but not sure if that is better.

Reply:
Good point! Thank you @exekias and @ruflin. In this case, I still feel like `bytes` makes more sense in the metric name instead of `data`(too broad). `bytes` in this case, comparing with `packets`, is identifying the type of this value instead of only being the unit postfix.